src/main/java/org/jabref/gui/preferences/table/TableTab.java

```diff
@@ -147,4 +146,8 @@ public void sortColumnDown() {
     public void addColumn() {
         viewModel.insertColumnInList();
     }
+
+    public TableView<MainTableColumnModel> getList() {
```

Comment:
I would give this a meaningful name

Reply:
Please do not use a getter for the TableView, instead inject the columnslist of the table into your new class RightClickMenu and work on that. We need to untangle all the callbacks, otherwise the classes are untestable.